packages/sit-onyx/src/styles/breakpoints.scss

```diff
@@ -23,3 +23,12 @@ $breakpoints: (
     @content;
   }
 }
+
+/**
+ * Same as `breakpoint` mixin but using container queries instead of media queries.
+ */
+@mixin container($min-max, $name, $offset: 0) {
```

Comment:
If we call this `container`, then the `breakpoint` mixin would make more sense to be called `media` or `screen`?

Reply:
I like it :) Also it reads very nice then:

```sass
breakpoints.screen(max, md)
breakpoints.container(max, md)
```